mpdfunk.py

```diff
@@ -0,0 +1,59 @@
+"""python-mpd2 library"""
+from mpd import MPDClient
+
+def mpdconnectionclose(client):
+    """Close network connection to MPD"""
+    client.close()
+    client.disconnect()
+
+def mpdsonginfo(client):
+    """Fetch the current song's title, artist, and fingerprint"""
+    title = client.currentsong()['title']
+    artist = client.currentsong()['artist']
+    fingerprint = client.readcomments(client.currentsong()['file'])['acoustid_fingerprint']
+    return title,artist,fingerprint
+
+def mpdalbumart(client):
+    """Dump the album art to disk"""
+    with open("cover.jpg", "wb") as file_open:
```

Comment:
Does this crash the bot if the art doesnt exist?

Reply:
Yes. I need to add error handling code here. Realistically, I want to write a test during bot launch that checks for album art across each song. I don't want a case of us playing a song without album art live.

Reply:
Use a try execption here then like so

```
    try:
        request = youtube.liveChatMessages().insert(
            part="snippet",
            body={
                "snippet": {
                    "liveChatId": live_chatid,
                    "type": "textMessageEvent",
                    "textMessageDetails": {
                    "messageText": "This is a test message from YoutubeProBot"
                    }
                }
            }
        )
        response = request.execute()
        print(response)
        logger.debug("Sent test command response to live stream")

    except request.HTTPError as request_httperror:
        logger.error(str(request_httperror))
```

Reply:
While we don't want to have a case where there is no art we shouldn't take down a stream at 3 in the morning just because one 3-4 min long song doesnt have a cover art.